packages/forms/src/templates/FieldTemplate.js

```diff
@@ -38,7 +39,7 @@ const CustomFieldTemplate = ({
 			<Label className="form-label" label={label} required={required} id={id} />}
 			{children}
 			{!hasLabelBefore && !isToggle && displayLabel &&
-			<Label label={label} required={required} id={id} className="control-label" />}
+			<Label label={label} required={required || schema.required} id={id} className="control-label" />}
```

Comment:
not sure to understand the difference between 
`"fullDatalist": {
       "required": true,
       "$ref": "#/definitions/datalist"
     }`

and `"required": [
      "select1"
    ]`